<launch>
   <arg name="config" default="$(find sas_space_explore)/cfg/params.yaml" />
   <env name="ROSCONSOLE_CONFIG_FILE"
          value="$(find sas_space_explore)/cfg/custom_rosconsole.conf"/>
   <node pkg="sas_space_explore" type="get_plan_server_node" name="get_plan_server_node" output="screen">
     <rosparam command="load" file="$(arg config)"/>
     <param name="angle_size" value="32" />
     <param name="heading_size" value="32" />
     <param name="allow_use_last_path" value="true" />
     <param name="allow_offset_distance" value="2" />
   </node>
   
    <node pkg="sas_space_explore" type="get_plan_client_node" name="get_plan_client_node" output="screen" >

    </node>


   <node pkg="rviz" type="rviz" name="rviz" args="-d $(find sas_space_explore)/rviz/sas_explore_node_viz.rviz"/>
   <node pkg="rqt_reconfigure"  type="rqt_reconfigure" name="reconfigure_gui"/>
   <node pkg="dynamic_reconfigure" type="dynparam" name="dynparam" args="load /get_plan_server_node $(find sas_space_explore)/cfg/dynamic_params.yaml"/>

</launch>
